lib/openxml/drawingml/properties/color_xform_alpha.rb

```diff
@@ -0,0 +1,19 @@
+module OpenXml
+  module DrawingML
+    module Properties
+      class ColorXformAlpha < ValueProperty
+        namespace :a
+        tag :alpha
+
+        def valid?
+          /^((100)|([0-9][0-9]?))(\.[0-9][0-9]?)?%$/ =~ value
```

Comment:
In **lib/openxml/drawingml/properties/color_xform_alpha_offset.rb**, You labeled `/^-?((100)|([0-9][0-9]?))(\.[0-9][0-9]?)?%$/ =~ value` as _ST_FixedPercentage, Part 1 - 20.1.10.24_. Is there no such label for this one?

This "positive percentage" regex is repeated in **color_xform_tint.rb** and **color_xform_shade.rb** too.

What you extracted these regexes to constants? Like:

```ruby
module OpenXml
  module DrawingML
     ST_FixedPercentage = `/^-?((100)|([0-9][0-9]?))(\.[0-9][0-9]?)?%$/` # §20.1.10.24
  end
end
```

If it's in the DrawingML namespace, it wouldn't clobber regexes defined in other openxml gems; but it should be available to Elements and Properties without you having to give its fully-qualified name.

Would that be a way of naming all these special regexes?

Reply:
Look like you name it in **dash_stop.rb** and **gradient_stop.rb** and **line_join_mitre.rb**

Reply:
☝️ I like this idea of extracting them into constants!